types/keychain/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
the module is a Node CJS module, I think it does not depend on 'dom'-runtime objects (so TS lib.dom). Please remove blocks with 'console' in tests or just replace them with something like:
```ts
err; // ExpectType KeychainError
```

---

Okay, will do this way